source/image-handler/image-request.ts

```diff
@@ -157,7 +157,11 @@ export class ImageRequest {
       const originalImage = await this.s3Client.getObject(imageLocation).promise();
       const imageBuffer = Buffer.from(originalImage.Body as Uint8Array);
 
-      if (originalImage.ContentType) {
+      if (key.endsWith(".heic")) {
```

Comment:
The library didn't even try to deal with HEIC images out of the box, so had to add this to make it recognize that image type and pass it through to the sharp library.

For the other image types it does bit sniffing of the actual file to figure out the format, but bit sniffing HEIC files it a lot more complicates than formats like JPEG or GIF, so we just hack looking at the uploaded file extension instead.